indico/web/client/js/utils/date.js

```diff
@@ -109,3 +110,25 @@ export function initialEndTime(end) {
     const endOfDay = moment().endOf('day');
     return end > endOfDay ? endOfDay : end;
 }
+
+/**
+ * Convert string date into date-agnostic datetime
+ * @param {String} dt - String datetime to transform
+ * @returns {moment.Moment} - Date-agnostic datetime
+ */
+export function getTime(dt) {
+    return moment(dt).set({year: 0, month: 0, day: 0});
+}
+
+/**
+ * Check whether there is a fully overlap between two pre-bookings
```

Comment:
Is it correct? `there is a fully overlap`? What about `full overlap` or sth like `Check whether two pre-bookings fully overlap`